TombEngine/Scripting/Internal/TEN/Objects/Lara/LaraObject.cpp

```diff
@@ -45,11 +45,48 @@ bool LaraObject::GetOnFire() const
 	return lara->BurnType != BurnType::None;
 }
 
+/// Set Poison with potency of poision
+// @function LaraObject:SetPoison
+// @tparam Potency Value 
+// @usage
+// Lara:SetPoison(100)
+void LaraObject::SetPoison(int potency = 0)
+{
+	auto* lara = GetLaraInfo(m_item);
+
+	lara->PoisonPotency = potency;
+}
+
+/// Get Poison potency of Lara
+// @function LaraObject:GetPoison
+// @usage
+// Lara:GetPoison()
+int LaraObject::GetPoison()
+{
+	auto* lara = GetLaraInfo(m_item);
+
+	return lara->PoisonPotency;
+}
+
+/// Remove Poison
+// @function Moveable:RemovePoison
+// @usage
+// Lara:RemovePoison()
+void LaraObject::RemovePoison()
```

Comment:
I doubt that we need dedicated  `RemovePoison` function, because in that case similar functions should be introduced for every pair of getter/setter that is already existing in API.